### DISCUSSION TITLE
1password-migration/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#38] [Comments]
# 1password-migration/

Almost exactly a year ago, I migrated from Chrome’s password manager to LastPass. Whilst this was luckily after the breach, the constant “oops, bad news” headlines destroyed my trust in LastPass. My renewal was coming up, but I migrated to 1Password instead: here’s how it went.

https://blog.jakelee.co.uk/1password-migration/

**Comment by Newtronic** [comment]
My experience was similar to yours: LastPass to 1Password.  I moved both my wife's and my account.  For my account, I was able to successfully export the CSV of passwords and imported them into 1Password.  Probably 500 accounts.  My wife's account appeared to export from LastPass, but the import wouldn't work.  Possibly a non-printable character in the wrong place?  I don't know.  Anyway, like you, I very carefully reviewed everything and let 1Password have the login to LastPass - worked flawlessly on her accounts.

It's been almost a year since I moved, but I remember finding some crazy workaround for LastPass export because I ran into the same problem you did.  I was really cussing out LastPass for an obvious attempt at vendor Lock In.

**Reply by JakeSteam** [reply]
Yeah, I wondered if one of my passwords was "too" hard to guess and broke their export... lucky 1Password have a direct import now, even if it is done via the very weird method of giving them your username & password & 2FA!
